### PR TITLE
program page redesign

### LINKED ISSUE #921: Program Pages: update template to match current design
mm_program_ 2x

Acceptance criteria:

- [x] match curent design
- [x] Hero image and blurb come from CMS
- [x] About the Program and Frequently asked questions come from CMS
- [x] contact us link comes from the CMS
- [x] header consistent with the rest of the site

Related issues:

- #922 : Courses Widget
- #923 : Faculty Profiles Widget 
- #917 : Sign Up / Get started modal

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #921 

#### What's this PR do?

This updates the design of the program page to match the mockups.

#### Where should the reviewer start?

I'd start by going to a program page and making sure it resembles the mockup. To get the full effect you'll need to specify a background image and fill in some example text in the description fields and so on.

Then I guess read through the HTML and CSS changes and make sure they make sense!

#### Screenshots (if appropriate)

about_program

faqs

### CODE REVIEW

File: cms/migrations/0015_auto_20160914_1932.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            model_name='programpage',
+            name='program',
+            field=models.OneToOneField(help_text='The program for this page', null=True, on_delete=django.db.models.deletion.SET_NULL, to='courses.Program'),
+        ),
+        migrations.AlterField(
+            model_name='programpage',
+            name='thumbnail_image',
+            field=models.ForeignKey(blank=True, help_text='Thumbnail size must be at least 690x530 pixels. Thumbnails are cropped down to this size, preserving aspect ratio.', null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to='wagtailimages.Image'),
+        ),
+    ]
```
Comment (aliceriot) [inline cms/migrations/0015_auto_20160914_1932.py]: I'm not sure what's going on here, but this is what happened when I did manage.py makemigrations.

**Comment by noisecapella** [comment]
Slightly outside this PR scope but can you change 'MicroMasters Portal' to 'MicroMasters' in header.html?

**Comment by noisecapella** [comment]
Actually do we have an issue for making the header look the same with the breadcrumbs? Is that covered under #921?

**Comment by aliceriot** [comment]
What do you mean by that exactly?

**Comment by noisecapella** [comment]
The `MicroMasters > Supply Chain Management` in the mockup

**Comment by roberthouse54** [comment]
@noisecapella @alice Don't worry about the breadcrumbs in the header issue. There is a late change request from marketing that means we may have to put the MicroMasters logo in the header, which I will probably do myself at the same time when I update the color palette (and copy) for the home page, the program page, and the app itself.

**Comment by aliceriot** [comment]
ok cool, sounds good to me

**Comment by aliceriot** [comment]
o just realized I forgot the little edx logo

**Comment by aliceriot** [comment]
do we have an asset for the micromasters logo?

**Comment by noisecapella** [comment]
Functionality looks good

**Comment by noisecapella** [comment]
@aliceriot @annagav added the asset with her PR

**Comment by aliceriot** [comment]
@noisecapella I just added the little images

**Comment by noisecapella** [comment]
I created the review, still not sure how the new github UI works.

File: cms/migrations/0015_auto_20160914_1932.py (hunk 1 above)
Comment (aliceriot) [inline cms/migrations/0015_auto_20160914_1932.py]: Cool, I was a little weirded out, haha

File: cms/templates/cms/program_page.html [hunk 2]
```diff
@@ -7,233 +7,97 @@
...
-
-                    <p>{{ page.description|richtext }}</p>
-
-
-                  </div>
-                  <div class="col-md-3 col-md-offset-1">
-
-                    <div class="sidebar-content">
-                      <h5>Contact Us</h5>
-                      {{ page.contact_us|richtext }}
```
Comment (aliceriot) [inline cms/templates/cms/program_page.html]: There's a 'contact us' `mailto` link down by the link to the program homepage

Comment (aliceriot) [inline cms/templates/cms/program_page.html]: I blew away all the old css though, which is why this was removed

Comment (noisecapella) [inline cms/templates/cms/program_page.html]: Can you add a migration to remove this field from the CMS?

Comment (aliceriot) [inline cms/templates/cms/program_page.html]: Oh you mean the `contact_us` field, gotcha

**Comment by aliceriot** [comment]
comments should be addressed!

**Comment by aliceriot** [comment]
@noisecapella is this g2g? I saw you switched the labels.

**Comment by noisecapella** [comment]
Sorry, still getting used to this UI

**Comment by aliceriot** [comment]
no worries

**Comment by aliceriot** [comment]
comments addressed

**Pull request merged**